.claude/commands/pr-unresolved.md

```
@@ -0,0 +1,108 @@
+# Get Unresolved PR Comments
+
+Get all unresolved review comments from a specific pull request using the GitHub CLI.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)